setup.py

```diff
@@ -32,7 +32,7 @@
         "neo4j>=1.7.0,<4.0.0",
         "neobolt>=1.7.0,<4.0.0",
         "policyuniverse>=1.1.0.0",
-        "python-dateutil>=2.7.0",
```

Comment:
Please add a comment linking to the botocore issue as the reason for the upper constraint.